backend/provisioner/service.go

```diff
@@ -33,7 +33,6 @@ type CommonProvisionerConfig struct {
 }
 
 type Config struct {
-	Bind               *url.URL `help:"Socket to bind to." default:"http://127.0.0.1:8893" env:"FTL_BIND"`
```

Comment:
How's this work in prod?

Reply:
The provisioner now listens for schema changes, it doesn't actually accept gRPC requests.